src/api/routers/participants/participantsCreation.ts

```diff
@@ -21,23 +35,44 @@ import {
   ParticipantCreationRequest,
 } from './participantClasses';
 
-export async function createParticipant(req: ParticipantRequest, res: Response) {
-  const participantRequest = ParticipantCreationRequest.parse(req.body);
+export async function validateParticipantCreationRequest(
```

Comment:
nit: I wonder if it makes more sense to not export this function. Makes it clear that it isn't used outside this file. Thoughts?